main.css

```diff
@@ -1,73 +1,81 @@
-#a-1 {
-    font-size: 15px;
-}
-
-#body {
+body {
     margin: 0;
     padding: 0;
     background-color: #eeeff2;
 }
 
-#btn-form {
-    font-family: Arial, Helvetica, sans-serif;
-    color: white;
-    font-size: 17px;
-    border: 3px;
-    border-bottom-color: black;
-    border-radius: 5px;
-    background: steelblue;
-    width: 100%;
-    height: 40px;
-}
+#sidebar {
+    /* float: right; */
+    position: absolute;
+    left: 1200px;
+    top: 150px;
 
-#cl-3 {
-    display: flex;
-    flex-wrap: wrap;
-  }
+}
 
-#col-2 {
+#videos {
+    /* float: left;
+  top: 120px; */
     position: absolute;
-    left: 700px;
-    top: 80px;
-    height: 200px;
-    width: 25%;
-    font-size: 20px;
-    font-family: Arial, Helvetica, sans-serif;
+    left: 25px;
+    top: 90px;
+}
+
+ul {
+    list-style-type: none;
+    height: 60px;
+    margin: 0;
+    padding: 5px;
     text-align: center;
+    display: inline;
+    width: 150%;
+
 }
 
-#col-3 {
-    position: absolute;
-    left: 1400px;
-    top: 30px;
-    height: 200px;
-    width: 25%;
-    font-size: 25px;
+li {
+    display: inline-block;
+}
+
+ul a {
+    text-decoration: none;
+    color: royalblue;
+    width: 100px;
+    display: block;
+    padding: 15px;
+    font-size: 17px;
+    font-family: Arial, Helvetica, sans-serif;
+
 }
 
+li img {
+    padding: 1px;
+}
 
-#corner-btn {
+#left {
     display: inline;
+    float: left;
 
 }
 
-#div-btn-form {
-    padding-left: 5%;
-    padding-right: 5%;
+#col-2 {
+    position: absolute;
+    left: 700px;
+    top: 80px;
```

Comment:
`HINT`

Using absolute values here to position the main content of the page may affect the display of the page. :thinking: Use `float` of `flex` instead.